src/ensembl/src/shared/components/question-button/QuestionButton.tsx

```diff
@@ -1,12 +1,21 @@
 import React, { useState } from 'react';
+import classNames from 'classnames';
 
 import Tooltip from 'src/shared/components/tooltip/Tooltip';
 import { ReactComponent as QuestionIcon } from './icon_question.svg';
 
-import styles from './QuestionButton.scss';
+import defaultStyles from './QuestionButton.scss';
+
+// Extra styling options based on where the button is located
+export enum QuestionButtonOption {
+  INPUT = 'in-input-field',
+  INLINE = 'inline'
+}
 
 type Props = {
   helpText: React.ReactNode;
+  styleOption?: QuestionButtonOption;
+  classNames?: { [key in QuestionButtonOption]?: string };
```

Comment:
I think there should be only one `className: string` rather than an object of strings. If ever the parent will want to pass an arbitrary class to the question button, it will likely be just a single class.